src/tools/rest-frontend/public/pages/main/auth/login.html

```diff
@@ -0,0 +1,70 @@
+<div class="row">
+    <div class="col-md-12">
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h4 class="panel-title">
+                    {{ 'APP.AUTH.LOGIN.PANEL.HEADING' | translate }}
+                </h4>
+            </div>
+            <div class="panel-body">
+                <form name="loginForm" class="form-horizontal">
+                    <div class="form-group has-feedback"
+					 ng-class="{'has-error': loginForm.username.$invalid && !loginForm.username.$pristine,
+								'has-success': loginForm.username.$valid && loginForm.username.$dirty}">
+                        <label for="username" class="control-label col-md-2">
+                            {{ 'APP.AUTH.LOGIN.LABEL.USERNAME' | translate }}
```

Comment:
Actually the translation module comes with an own directive that pretty much copies the behavior of ng-bind: `<ANY translate="TRANSLATION_ID"></ANY>`
The only advantage is that you can set default-/fallback-texts on page load, but I haven't really experienced that problem yet. Changing all of the languages variables now could be a bit cumbersome. :(

But we can definitely note it as possible improvement in the future!